src/Sarif/Readers/DelegatingStream.cs

```diff
@@ -30,26 +35,20 @@ public override long Position
             }
             get => stream.Position;
         }
-        public override void Flush()
-        {
-            stream.Flush();
-        }
-        public override int Read(byte[] buffer, int offset, int count)
-        {
-            return stream.Read(buffer, offset, count);
-        }
-        public override long Seek(long offset, SeekOrigin origin)
-        {
-            return stream.Seek(offset, origin);
-        }
-        public override void SetLength(long value)
-        {
-            stream.SetLength(value);
-        }
-        public override void Write(byte[] buffer, int offset, int count)
-        {
-            stream.Write(buffer, offset, count);
-        }
+
+        public override void Flush() => stream.Flush();
+
+        public override int Read(byte[] buffer, int offset, int count) => stream.Read(buffer, offset, count);
+
+        public override long Seek(long offset, SeekOrigin origin) => stream.Seek(offset, origin);
+
+        public override void SetLength(long value) => stream.SetLength(value);
+
+        public override void Write(byte[] buffer, int offset, int count) => stream.Write(buffer, offset, count);
+
+        /// <summary>
+        /// Calls the dispose method on the wrapped stream.
```

Comment:
>Calls the dispose method on the wrapped stream. [](http://example.com/codeflow?start=12&length=47)

? This comment provides no value over reading the code or reading the name 'DisposeUnderlyingStream. Please think of something additionally useful to say or delete the comment.

You will hear this from me persistently: don't author comments that are a simple recapitulation of what's already obvious in code. A comment should provide additional value.